tests/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../.eslintrc",
```

Comment:
No need for this, the child configuration file automatically inherits features from the parent depending on the directory _eslint_ is being executed from

Reply:
Awesome, fixed